system/jlib/jlog.ipp

```diff
@@ -253,6 +253,24 @@ private:
     bool                      localFlag;
 };
 
+    
+class WUIDLogMsgFilter : public CLogMsgFilter
+{
+public:
+    WUIDLogMsgFilter(const char * _wuid) : wuid(_wuid) {}
+    WUIDLogMsgFilter(MemoryBuffer & in) { in.read(wuid);}
+    WUIDLogMsgFilter(IPropertyTree * tree) { wuid.set(tree->queryProp("@wuid")); }
+
+    bool                      includeMessage(const LogMsg & msg) const { return false; }
+    bool                      mayIncludeCategory(const LogMsgCategory & cat) const { return true; }
+    unsigned                  queryAudienceMask() const { return MSGAUD_all; }
+    unsigned                  queryClassMask() const { return MSGCLS_all; }
+    LogMsgDetail              queryMaxDetail() const { return TopDetail; }
+    void                      serialize(MemoryBuffer & out, bool preserveLocal) const { out.append(MSGFILTER_pid).append(wuid); }
```

Comment:
>MSGFILTER_pid

I think you need a new enumerated value here? e.g. MSGFILTER_wuid

Reply:
Removed